app/Filament/Resources/HubResource/RelationManagers/GamesRelationManager.php

```diff
@@ -0,0 +1,212 @@
+                    ->numeric()
+                    ->sortable()
+                    ->alignEnd(),
+            ])
+            ->filters([
+                Tables\Filters\SelectFilter::make('system')
+                    ->relationship('system', 'Name'),
+
+                Tables\Filters\TernaryFilter::make('achievements_published')
+                    ->label('Has core set')
+                    ->placeholder('Any')
+                    ->trueLabel('Yes')
+                    ->falseLabel('No')
+                    ->queries(
+                        true: fn (Builder $query): Builder => $query->where('achievements_published', '>=', 6),
+                        false: fn (Builder $query): Builder => $query->where('achievements_published', '<', 6),
+                        blank: fn (Builder $query): Builder => $query,
+                    ),
+            ])
+            ->headerActions([
+                // TODO after dropping GameAlternatives, try to use native attach
+                Tables\Actions\Action::make('add')
+                    ->label('Add games')
+                    ->form([
+                        Forms\Components\TextInput::make('game_ids_csv')
+                            ->label('Game IDs (CSV)')
+                            ->placeholder('729,2204,3987,53')
+                            ->helperText('Use a comma-separated list of game IDs.'),
+
+                        Forms\Components\Select::make('game_ids')
+                            ->label('Games')
+                            ->multiple()
+                            ->options(function () {
+                                return Game::whereNotIn('ID', $this->getOwnerRecord()->games->pluck('ID'))
+                                    ->where('ConsoleID', '!=', System::Hubs)
+                                    ->limit(50)
+                                    ->get()
+                                    ->mapWithKeys(fn ($game) => [$game->ID => "[{$game->ID}] {$game->Title}"]);
+                            })
+                            ->searchable()
+                            ->getSearchResultsUsing(function (string $search) {
+                                return Game::whereNotIn('ID', $this->getOwnerRecord()->games->pluck('ID'))
+                                    ->where('ConsoleID', '!=', System::Hubs)
+                                    ->where(function ($query) use ($search) {
+                                        $query->where('ID', 'LIKE', "%{$search}%")
+                                            ->orWhere('Title', 'LIKE', "%{$search}%");
+                                    })
+                                    ->limit(50)
+                                    ->get()
+                                    ->mapWithKeys(fn ($game) => [$game->ID => "[{$game->ID}] {$game->Title}"]);
+                            })
+                            ->helperText('... or search and select games to add.'),
```

Comment:
In latest, when the user starts typing in one field, the other will become disabled.

Reply:
This is better, but because the disabling happens on lost focus (I'm assuming), typing into the first field and then clicking on the dropdown field opens the dropdown before it gets disabled, then you're stuck in a state with a dropdown that can't be closed.

Reply:
It's a quirk of these controls being Livewire 😕
The server is making a roundtrip on field change to determine if the other field should be disabled.

I've made a slight adjustment in latest that should at least resolve this issue. Now, when the user provides input in one of the fields, it will be set to both disabled and hidden:

![Screenshot 2024-12-21 at 2 03 29 PM](https://github.com/user-attachments/assets/83750a1d-c6a4-4ba8-ab7d-2d753494c311)